### DISCUSSION TITLE
MethodError - REopt.varElectricTariff111

### DISCUSSION THREAD

**Discussion by d-rao** [NREL/REopt-API-Analysis#150] [V3 Transition]
I seem to be getting the above error as I post to the V3 API. Here is the run_uuid: 9322ced1-f7a0-4f6f-9bc7-a3c574bbfc78. I've attached the latest input and output files for reference too. Thanks for the help!

[my_results.json](https://github.com/NREL/REopt-Analysis-Scripts/files/14829022/my_results.json)
[input_Reopt_20240329.json](https://github.com/NREL/REopt-Analysis-Scripts/files/14829024/input_Reopt_20240329.json)

**Comment by adfarth** [comment]
This error was due to a bug in how REopt handles the input `export_rate_beyond_net_metering_limit`. The fix for this will be live shortly, and users can remove the input if not needed in the meantime.
